pf_description/CMakeLists.txt

```diff
@@ -1,27 +1,15 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.7)
```

Comment:
@wawanbreton this package has no cpp code so CMakeLists.txt is not needed. Please take a look at https://github.com/PepperlFuchs/pf_lidar_ros_driver/pull/83/commits/9476ad63b870801110c965d7b480135093a04f58 in https://github.com/PepperlFuchs/pf_lidar_ros_driver/pull/83

Reply:
Isn't it required to properly deploy the target files ?

Reply:
what do you mean? for python only packages, there is a [setup.py](https://github.com/PepperlFuchs/pf_lidar_ros_driver/blob/porting-ros2/pf_description/setup.py)

Reply:
But there is no Python code either :D so my answer is yes, something is required, either a CMakeLists.txt or a setup.py, which contains the deployment instructions

Reply:
launch files ;)

Reply:
Indeed, but they can still be deployed with a CMakeLists.txt, although this is quite nonsense :)